content/en/service_owners_guide/run.md

```diff
@@ -0,0 +1,116 @@
+
+## Fielding technical issues
+
+Due to its PaaS structure, there is only a limited area of troubleshooting that exists for Datadog.  To help identify issues in the host Agent, use the provided Datadog [command][3] *`datadog-agent status`*.  This command will report granular, specific, and actionable information that identifies areas to be addressed. Additionally, memorize and use the *`datadog-agent flare`* [command][4] to quickly surface issues that need to be addressed by Datadog Support.
+
+**Recommendation**: 
+Utilize the `status` and `flare` commands from day one.
+
+## Service Administration Tasks
+
+Like all other enterprise software, ongoing maintenance tasks need to be well-organized and adherent to your local policies. The following outline common ongoing tasks:
+
+### Usage Monitoring
+
+It is important to keep an eye on consumption, and equally so, to adopt the tools provided for this purpose. Datadog provides an [estimated usage metrics][5] dashboard that can be utilized to create the foundation of this capability. There are also out of the box dashboards for visualizing the estimated usage across all of your [logs, metrics, and traces][6]. 
+
+### Deploy Dashboards and monitors
+
+After becoming familiar with Datadog, your users may start asking for refinements and adjustments, particularly of the items they interact with the most; [dashboards][7] and [monitors][8]. The dashboards, monitors, SLOs, and other content objects are designed for iterative development.  At their core, these items are written in JSON. They can be cloned, exported, modified, imported, and stored as flat files. Additionally, there is a [terraform provider][9] available, and a [dashboards API][10] that can interact with and create dashboards.  
+
+When creating dashboards, there are many things to consider, but the focus should be on what content to display rather than on the construction process of the dashboard itself. This creative process is embraced within Datadog dashboards, and it is supported through a series of offerings and materials. Each of the dashboards included within the 700+ integrations is a value-added template for monitoring for the relative technology. In the out-of-the-box dashboards, notice Datadog's experience and prescriptive model for observability at the individual product level.  
+
+**Recommendations:**  
+
+- Determine the purpose of the dashboard you're creating.   
+- Set Monitors for Datadog usage built off the [Estimated Usage metrics][6].  
+- Create [anomaly or change monitors][11] on these same estimated usage metrics to send alerts when your Datadog usage spikes.    
+- [Re-use and clone][12] other dashboards to save time.  
+- Utilize [OOTB dashboards][13] to govern consumption.
+
+A common OOTB Dashboard is the AWS EC2 Overview Dashboard:
+
+{{< img src="/service_owners_guide/ec2_overview.png" alt="AWS EC2 Overview Dashboard" style="width:90%;">}}
+
+### API key rotation 
```

Comment:
Could be good to add our [API key method in here](https://docs.datadoghq.com/api/latest/key-management/)

Reply:
added